docker/set_up_ubuntu.sh

```diff
@@ -24,18 +24,14 @@ apt-get install -y gcc apt-transport-https build-essential python-dev \
 
 # Stomp on the bash prompt with something more useful for development.
 cat > /etc/bash.bashrc <<EOF
-# Get the name for the current uid and drop error messages.
-MYUSER="\$(id -u -n 2> /dev/null)"
-
-# If id -u -n 2 didn't return a 0 exit code, then it's because there is
-# no name for that uid in the container. So set it to "you".
-if [[ "\$?" != "0" ]]; then
-    MYUSER="you"
-fi
```

Comment:
This was a bunch of mumbo jumbo to deal with setting the userid part of the prompt so that it was helpful for people. So if you used the `as_me.sh` script, you'd get `you` and if you didn't, it'd be `app`.

We don't need to worry about that anymore. We'll always use `app` now.